src/modules/household/components/householdActions/SelectClientsStep.tsx

```diff
@@ -70,11 +74,13 @@ const SelectClientsStep = ({
       <Paper>
         <GenericTable<HouseholdClientFieldsFragment>
           rows={donorHouseholdMembers}
+          rowName={(row) => clientBriefName(row.client)}
           columns={MANAGE_HOUSEHOLD_COLUMNS}
           selectable={'checkbox'}
           selected={selectedClientIds}
           onChangeSelectedRowIds={setSelectedClientIds}
           isRowSelectable={isRowSelectable}
+          getRowSelectDisabledReason={getRowSelectDisabledReason}
```

Comment:
This is getting into revisiting design, so feel free to ignore, but just want to throw it out there. I didn't think of it when reviewing the designs, but another approach would be to just exclude the HoH(s) from this table. It seems potentially more straightforward, because the user is only faced with the options they _can_ select, rather than also including options they can't select (with disabled controls and hover actions to understand why). It seems like less cognitive load to me.